apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTestRunner.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef AUDIO_WORKLOAD_TEST_RUNNER_H
+#define AUDIO_WORKLOAD_TEST_RUNNER_H
+
+#include <string>
+#include <chrono>
+#include <atomic>
+#include <memory>
+#include "AudioWorkloadTest.h"
+
+class AudioWorkloadTestRunner {
+public:
+    AudioWorkloadTestRunner();
+    ~AudioWorkloadTestRunner();
+
+    int32_t start(
+            int32_t numCallbacks,
```

Comment:
Using numCallbacks for the duration will make it hard to run a test for N seconds.
We may need that if a caller is trying to limit the time spent on each test.
Maybe pass a duration in seconds and calculate the numCallbacks after you get the sample Rate and framesPerBurst.